Project.toml

```diff
@@ -12,7 +12,7 @@ MuladdMacro = "46d2c3a1-f734-5fdb-9937-b9b9aeba4221"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 
 [compat]
-ERFA = "^0.4"
+ERFA = "^0.6"
```

Comment:
Why not 0.4 and 0.5?  Also, the caret is redundant, it's the default

Reply:
because another dependency required at least 0.6
the caret is copying the existent style